### ISSUE TITLE
Third-party strings not translatable 

### ISSUE THREAD

**Issue by Loosie94** [bytefury/crater#235]
**Is your feature request related to a problem? Please describe.**
I've just translated the Dutch version for crater v3.1.1. Now i see a lot of strings like: 'There are no matching rows' and 'This year' still in English. Probably because they are third party. Is there a way we can translate those as well? That would be really nice.

**Describe the solution you'd like**
Pass (new) variables in this place to translate them

**Describe alternatives you've considered**
None

**Comment by mohitpanjwani** [comment]
Good catch! I personally use the English version so we tend to miss these kinds of small details. 

Thanks for heads up, we'll make them translatable 👍

**Comment by vdvcoder** [comment]
@Loosie94 is it possible that i can have a copy of your Dutch translation please?

Thanks in advance!

**Comment by Loosie94** [comment]
@vdvcoder It has already been merged with the latest Crater vesion (3.1.1). But you better check my fork for an improved version: https://github.com/Loosie94/crater/blob/master/resources/assets/js/plugins/nl.json

I'll make a new pull-request as well.
